swix/core/Microsoft.VisualStudio.VC.Ide.TestAdapterForGoogleTest.swr

```diff
@@ -0,0 +1,11 @@
+use vs
+
+package name=Microsoft.VisualStudio.VC.Ide.TestAdapterForGoogleTest
+        version=$(Version)
+        vs.package.type=vsix
+        vs.package.installSize=22265856
```

Comment:
We should note in the Release steps to keep this up-to-date.

Reply:
Will do!